data-spring-jdbc/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 plugins {
   `java-library`
   id("org.springframework.boot") version "2.6.3"
-  id("io.spring.dependency-management") version "1.0.11.RELEASE"
+  id("io.spring.dependency-management") version "1.0.12.RELEASE"
```

Comment:
Pretty sure spring version is unrelated

Reply:
You're right, I did this without checking what I was updating. I just searched for "1.0.11"